src/pages/Reader.tsx

```diff
@@ -135,6 +160,7 @@ class Reader extends React.Component<any, State> {
       ...curState,
       readerSettings: newReaderSettings
     }));
+    localStorage.setItem("carrotSettings", JSON.stringify(newReaderSettings))

```

Comment:
not necessarily needed but possibly check that newReaderSettings doesn't have any weird values before dumping it into localstorage?

Reply:
not too particularly worried about dumping into localstorage, more worried about getting because its hard to modify react state in production

Reply:
ok not too important ig